sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -111,9 +117,11 @@ internal static bool IsJson(string input)
                 return false;
             }
 
-            input = input.Trim();
-            return (input.StartsWith("{", StringComparison.OrdinalIgnoreCase) && input.EndsWith("}", StringComparison.OrdinalIgnoreCase))
-                || (input.StartsWith("[", StringComparison.OrdinalIgnoreCase) && input.EndsWith("]", StringComparison.OrdinalIgnoreCase));
+            // try parsing input to json object
+            using (var jsonDoc = JsonDocument.Parse(input))
```

Comment:
JsonDocument.Parse doesn't return null, it throws when document is invalid: https://dotnetfiddle.net/s57G0p

Reply:
Simplified this and returned true